### DISCUSSION TITLE
字幕只有一个字的情况下识别不出来？

### DISCUSSION THREAD

**Discussion by dzy49** [SWHL/RapidVideOCR#57] [Q&A] [answered]
感谢作者！
其他情况下都很不错 但是字幕只有单字的情况下大概率识别不出来？
VSF 是有识别到并且输出了RGB图片的

**Comment by SWHL** [comment]
给几张示例图片呗

**Comment by dzy49** [comment]
![0_19_11_301__0_19_11_860_0000005761280014412800720](https://github.com/SWHL/RapidVideOCR/assets/32559093/53742f50-2b4b-4dab-93d2-56a2ec12fa90)
![0_19_37_503__0_19_37_942_0000005761280014412800720](https://github.com/SWHL/RapidVideOCR/assets/32559093/c71e972c-0844-44dc-9580-c0316035b849)

**Comment by SWHL** [comment, marked as answer]
这个丢失是因为识别到的文本置信度太低，导致被过滤了。可以添加`text_score=0.4`参数来调节。示例代码如下：
```python
from rapid_videocr import RapidVideOCR

extractor = RapidVideOCR(is_concat_rec=True, is_print_console=False, text_score=0.4)

rgb_dir = "tiny"
save_dir = "outputs"
save_name = "a"

# outputs/a.srt  outputs/a.txt
extractor(rgb_dir, save_dir, save_name=save_name)
```

详细参数，可参考[RapidOCR docs](https://rapidai.github.io/RapidOCRDocs/docs/blog/01_optim_tutorial/)
